bot/exts/fun/fun.py

```diff
@@ -51,6 +51,21 @@ def _get_random_die() -> str:
         die_name = f"dice_{random.randint(1, 6)}"
         return getattr(Emojis, die_name)
 
+    @staticmethod
+    async def _clean_fun_cog_text(ctx: Context, text: str, conversion_func: Callable[[str], str]) -> None:
```

Comment:
This function should not do the sending part, let it just clean the text & embed and return them as a tuple.

Also, the `fun_cog` in the function name is a redundant since it's a **private function** of the `Fun` cog.

Reply:
Hey!
Thanks for the feedback. I've changed the name and the function now returns said tuple.
I'm struggling to put the BadArgument check in `self._clean_text` for some reason. I've now got the raise BadArgument added though and it's working. But I don't like the repeating of it.